### ISSUE TITLE
Incorrect progress reporting in case of errors during downloadd

### ISSUE THREAD

**Issue by sjev** [ranaroussi/yfinance#207]
I've tried to download all SPY holding with 

```
data = yf.download(tickers=holdings.Ticker.to_list(),
                  start = start_date,
                  auto_adjust=True,
                  threads=True)
```
BRK.B and BF.B failed, this apparently messed up the progress bar: 

```
[*********************100%***********************]  505 of 505 completed

2 Failed downloads:
- BRK.B: No data found, symbol may be delisted
- BF.B: 1d data not available for startTime=-2208988800 and endTime=1583062138. Only 100 years worth of day granularity data are allowed to be fetched per request.
[****************************************179%******************************************]  903 of 505 completed
```

**Comment by martimfj** [comment]
Can you provide a minimal reproducible example?

ref: https://stackoverflow.com/help/minimal-reproducible-example

**Comment by rdou** [comment]
I saw the same issue, please try the below minimal reproducible example.

```
import yfinance as yf

stock_df = yf.download(tickers = 'BF.B', period = 'max')

Error message:
[*********************100%***********************]  1 of 1 completed

1 Failed download:
- BF.B: 1d data not available for startTime=-2208988800 and endTime=1615396753. Only 100 years worth of day granularity data are allowed to be fetched per request.
```

**Comment by silvavn** [comment]
The current version shows 100% Progress bar, and the error.
What did you expect to see?
